.github/workflows/check-ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
     branches: [ main ]
```

Comment:
Shouldn't we run checks on PRs to release branches as well? They should pass tests before merging, even if it's being cherry-picked from main which has a passing test (just because it passes on main doesn't mean it passes on release/*)

Reply:
We don't have long-lived release branches, and aren't really supporting PRs into short-lived release branches. We should be empowered to re-write history on our short-lived release branches if necessary; so just pushes to them should be OK IMO. It's really only the release manager that should be interacting directly w/ the release branch for now.